src/space_flight/pipelines/data_engineering/nodes.py

```diff
@@ -101,3 +101,37 @@ def create_master_table(
     master_table = with_companies.drop(["shuttle_id", "company_id"], axis=1)
     master_table = master_table.dropna()
     return master_table
+
+
+# Implementation of a sorting algorithm
+# Source: https://bosker.wordpress.com/2011/04/29/the-worst-algorithm-in-the-world/
+def _slowsort(values, i, j):
```

Comment:
Bad implementation - let me know when you have fixed it!!!!!! 
![image](https://user-images.githubusercontent.com/39539935/97363774-2dd73780-18f7-11eb-99aa-5c09b8284d7c.png)

Reply:
i'll reply too

Reply:
Ready?

Reply:
another comment

Reply:
This is pure !@#$!@#$!@$. My dog writes better code than this.   
![image](https://user-images.githubusercontent.com/39539935/97363972-75f65a00-18f7-11eb-83af-0659f8273151.png)

Reply:
does it tho?

Reply:
That's !@#$!@#$!@$. This is a great algorithm.  Also   
![image](https://user-images.githubusercontent.com/58553/97364146-b6ee6e80-18f7-11eb-8c4e-0c12899e0549.png)